### PR TITLE
Minimal fixes to documentation of Context and Host

### PR DESCRIPTION
**Pull request by rrodewald**
- Documented default value of unpackWARs attribute
- Tagged all boolean values als code
- Punctuation

### CODE REVIEW

File: webapps/docs/config/host.xml [hunk 1]
```diff
@@ -302,7 +302,7 @@
         placed in the <code>appBase</code> directory as web application
         archive (WAR) files to be unpacked into a corresponding disk directory
         structure, <code>false</code> to run such web applications directly
-        from a WAR file. See
+        from a WAR file. The flag's value defaults to <code>true</code>. See
```
Comment (michael-o) [inline webapps/docs/config/host.xml:305]: The default is...

**Review by michael-o — CHANGES_REQUESTED** [review]

File: webapps/docs/config/host.xml (hunk 1 above)
Comment (rrodewald) [reply webapps/docs/config/host.xml:305]: Shall I change this for all other occurences of the same wording?

**Review by michael-o — APPROVED** [review]

**Comment by kkolinko** [comment]
Reviewing...
1. context.xml L187  `s/docBase/<code>docBase</code>/ `as well
2. I am more used to "The default value is" rather than "The default is".
I wonder what wording is better. Tomcat documentation uses both.

**Comment by rrodewald** [comment]
> Reviewing...
> 
>     1. context.xml L187  `s/docBase/<code>docBase</code>/ `as well
> 
>     2. I am more used to "The default value is" rather than "The default is".
>        I wonder what wording is better. Tomcat documentation uses both.

1. Done
2. I am no native speaker so I do whatever I'm told to do...

**Comment by ChristopherSchultz** [comment]
To a native English speaker, either one is completely understandable. 
"The default value is ___" is slightly more clear, but I would actually 
defer to NON-native English speakers because it may be more 
understandable to them one way or the other.

I'm happy to listen to the Russian :)

**Comment by rrodewald** [comment]
If we want to standardize the wording for default values, I would suggest to open a new pull request for this, so that the main purpose of the present pull request (documentation of the default value of unpackWARs) can be considered as done.

Here is an incomplete list of wordings found:
- Defaults to ...
- Default [value] is ...
- The default [value] is ...

**Comment by markt-asf** [comment]
Thanks for the PR. +1 tp a new PR to make the default wording consistent.
Applied manually so I could a) add a change log entry and b) broadly maintain the line length limit of 80 chars for xml files.

**Pull request closed without merging**